### DISCUSSION TITLE
想问一下这个软件是否会被小蓝熊（easy anti-cheat）或者Ricochet识别为作弊插件

### DISCUSSION THREAD

**Discussion by li-jiaming-softbank** [MaaAssistantArknights/MaaAssistantArknights#1544] [Q&A] [answered]
想问一下这个软件是否会被小蓝熊（easy anti-cheat）或者Ricochet识别为作弊插件？

最近听说因为在玩游戏时后台还在运行脚本或者自动化程序，导致Apex或者COD战区等账户被封禁的事件时有发生。考虑到之前只有COD使用的Ricochet通过内核级反作弊驱动程序来扫描后台运行的程序（这会导致部分脚本被识别为作弊软件而造成账户被封禁），而最近Apex在最新的反作弊更新中也开始使用内核级安全驱动程序来扫描后台，所以我想问一下如果在玩这些使用了这类反作弊程序的游戏时如果没有关闭MAA是否有可能造成类似的误判情况导致被封禁。谢谢。

**Comment by ABA2396** [comment, marked as answer]
应该不会，这几天Apex和Maa方舟一起开也没什么问题，也没听说群友有因为这个被封的

**Comment by bingyunhe** [comment]
就apex那个废物蓝熊，检测不出来的
